server/routes/alpha/alphaRouter.ts

```diff
@@ -0,0 +1,10 @@
+import { Router } from 'express'
+import activityListRoutes from './activity-list/activityListRoutes'
+import { Services } from '../../services'
+
+export default function alphaRouter(services: Services): Router {
+  const router = Router({ mergeParams: true })
+  router.use('/activity-list', activityListRoutes(services))
+  // Add more routes here
+  return router
+}
```

Comment:
I'm perfectly ok with linking one of the homepage tiles to your routes for now? And renaming the tile to be `Activity list`. They don't point anywhere at the moment.

Reply:
Resolving in response to comment below